doc/release-notes.md

```diff
@@ -293,6 +293,24 @@ if this is 1.
 - `-datacarriersize=n` : Maximum size, in bytes, we consider acceptable for
 "data carrier" outputs.
 
+BIP 66: strict DER encoding for signatures
+------------------------------------------
+
+Bitcoin Core 0.10 implements BIP 66, which introduces block version 3, and a new
+consensus rule, which prohibits non-DER signatures. Such transactions have been
+non-standard since Bitcoin v0.8.0 (released in February 2013), but were
+technically still permitted inside blocks.
+
+This change breaks the dependency on OpenSSL's signature parsing, and is
+required if implementations would want to remove all of OpenSSL from the
+consensus code.
```

Comment:
I would mention that OpenSSL does not guarantee consensus-compatible behaviour here too.

"This change breaks the dependency on implementation-specific details in OpenSSL's signature parsing, avoids depending on OpenSSL maintaining those details to be consensus-compatible (which they do not guarantee), and is required if implementations would want to remove all of OpenSSL from the consensus code."